### PR TITLE
Monitor update rtypes and samples

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/tests/test_base_exporter.py [hunk 1]
```diff
@@ -73,11 +73,6 @@ def test_constructor(self):
         self.assertEqual(base.storage._retention_period, 604800)
         self.assertEqual(base._timeout, 10)
 
-    def test_constructor_wrong_options(self):
```
Comment (rakshith91) [inline sdk/monitor/azure-opentelemetry-exporter-azuremonitor/tests/test_base_exporter.py:76]: we do need to support kwargs - so this behavior is against the guidelines

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/export/trace/_exporter.py [hunk 2]
```diff
@@ -53,8 +54,8 @@ def export(self, spans: Sequence[Span]) -> SpanExportResult:
             logger.exception("Exception occurred while exporting the data.")
             return get_trace_export_result(ExportResult.FAILED_NOT_RETRYABLE)
 
-    def shutdown(self):
-        """Shuts down the exporter.
+    def close(self) -> None:
```
Comment (lzchen) [inline sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/export/trace/_exporter.py:57]: `shutdown` is the method that is inherited from [SpanExporter](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py#L62) in Opentelemetry. We should stick to the same method name.

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/traces/README.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ These code samples show common champion scenario operations with the AzureMonito
 * Server: [sample_server.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/traces/sample_server.py)
 * Azure Service Bus Send: [sample_servicebus_send.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/traces/sample_servicebus_send.py)
 * Azure Service Bus Receive: [sample_servicebus_receive.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/traces/sample_servicebus_receive.py)
+* Azure Storage Blob Create Container: [sample_storage.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/traces/sample_storage.py)
```
Comment (lzchen) [inline sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/traces/README.md:18]: Nit: Please sort alphabetically.

**Review by lzchen — APPROVED** [review]

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/export/trace/_exporter.py [hunk 4]
```diff
@@ -53,8 +54,8 @@ def export(self, spans: Sequence[Span]) -> SpanExportResult:
...
+        """Closes the exporter.
```
Comment (rakshith91) [inline sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/export/trace/_exporter.py:58]:
```suggestion
    def shutdown(self) -> None:
        """Shuts down the exporter.
```

**Pull request merged**